doc/controls/SafeArea.md

```diff
@@ -39,26 +44,20 @@ Mode|`InsetMode`|Gets or sets whether the `SafeArea` insets will be applied to t
 
 The `InsetMask` enum can represent a single bound or it can be combined with multiple values as shown in the above example XAML. `InsetMask` has the following available values:
```

Comment:
```suggestion
The `InsetMask` enum can represent a single edge/side or it can be composed of multiple values (eg: `InsetMask="Left, Right"`). `InsetMask` has the following available values:
```